### ISSUE TITLE
VMAlert oauth2 error

### ISSUE THREAD

**Issue by vaishutin** [VictoriaMetrics/VictoriaMetrics#6471]
### Describe the bug

Hello. I use vm-operator last version and create VMAlert manifest with values:
```yaml
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMAlert
metadata:
  name: demo
  namespace: grafana
spec:
  datasource:
    url: "http://x.x.x.x:8481/select/0/prometheus"
  notifiers:
  - url: "https://x.x.x.x/monitoring"
    oauth2:
      token_url: https://x.x.x.x/auth/realms/xxx/protocol/openid-connect/token
      client_id:
        configMap:
          name: vmalert-token-cm
          key: client_id
      client_secret:
        name: vmalert-token
        key: client_secret
  evaluationInterval: "30s"
  selectAllByDefault: true
```

POD no started and log output:
```
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:12	build version: vmalert-20231116-194705-tags-v1.95.1-0-g354563393
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:13	command-line flags
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -datasource.url="secret"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -evaluationInterval="30s"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -httpListenAddr=":8080"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -notifier.oauth2.clientID="alert-monitoring"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -notifier.oauth2.clientSecretFile="secret"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -notifier.oauth2.tokenUrl="secret"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -notifier.url="secret"
2024-06-12T19:26:37.093Z	info	VictoriaMetrics/lib/logger/flag.go:20	  -rule="/etc/vmalert/config/vm-demo-rulefiles-0/*.yaml"
2024-06-12T19:26:37.093Z	fatal	VictoriaMetrics/app/vmalert/main.go:163	failed to init: failed to init notifier: failed to create notifier from flag values: failed to configure auth: ClientSecret and ClientSecretFile cannot be set simultaneously
```

### To Reproduce

1. Deploy VMOperator with Helm 0.32.0
2. Create manifest VMAlert with values from error
3. Open pod logs

### Version

vmalert-20231116-194705-tags-v1.95.1-0-g354563393

**Comment by hagen1778** [comment]
@vaishutin the [fix](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6478) was merged and will be included into the next release.

**Comment by valyala** [comment]
FYI, this issue has been fixed in [vmalert v1.102.0-rc2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.0-rc2).

**Comment by valyala** [comment]
FYI, this bug has been fixed in [vmalert v1.93.16 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.16).

**Comment by valyala** [comment]
FYI, this issue has been fixed in [vmalert v1.97.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.97.6).
